pkg/auth/auth_test.go

```diff
@@ -0,0 +1,129 @@
+package auth
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+const cred = `
+{
+    "cloud":"AzurePublicCloud",
+    "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
```

Comment:
Is this a real tenantId? If both tenantId and subscriptionId are faked. please add a comment in the same line saying they are faked strings.

Reply:
The old code used to use a real tenant and subscription IDs! Planning to get rid of them in the next phase.

Reply:
Replaced.